nav.adoc

```diff
@@ -27,16 +28,18 @@
 //** xref:installation/apply-ilmt-kubernetes.html[Apply ILMT Annotation to Kubernetes Pods]
 * xref:runtime-services-modeling/runtime-services-modeling.html[Runtime Services Modeling]
 ** xref:runtime-services-modeling/decisions-with-dmn.html[Decisions with DMN]
-// ** xref:runtime-services-modeling/decision-orchestration-with-bpmn.html[Decision Orchestration with BPMN]
-** xref:runtime-services-modeling/deploying-services.html[Deploying Services]
+** xref:runtime-services-modeling/workflow-with-bpmn.html[Workflow with BPMN]
```

Comment:
Workflow with BPMN - > Not sure on this. Can this be renamed to 'Workflow for the BPMN'?

Reply:
That doesn't really make sense. Maybe Workflow for BPMN or Workflow using BPMN?

Reply:
Yes, looks fine